kod/util/system.kod

```diff
@@ -6574,6 +6574,22 @@ messages:
       }
       return total;
    }
+
+   CleanAllBadMail()
+   "Repeatedly calls CleanBadMail on all users until no bad mails remain."
```

Comment:
This is pretty much guaranteed to hit the Blakod interpreter limit on the number of instructions allowed in a single top-level call that prevents infinite loops.  I'll have to figure out the best way to do this, i.e. do I need to bring the server down and do it locally on my machine with a copy of the saved game?

Reply:
Based on this, I removed that function since I really don't like the idea of the operation aborting itself partway.